Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Demetia ###################
+	option = { 
+		name = BTWK_story_event.0104.h
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_demetia
+					AND = {
+						is_primary_heir_of = title:k_demetia.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_demetia
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_demetia.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+}
+
+BTWK_story_event.0105	= { # Guardian Selection Event
+
+# SELECTION OF GUARDIANS QUITE IRRELEVANT IN THE STORY. COULD GIVE PLAYER THE OPTION TO CHOOSE FROM REMAINING SUITORS, OR LEAVE IT AS CANON.
+# IF HAVE LIST OF ALL PRESENT, ADD TRUCES BETWEEN EACH KING TITLE HOLDER. 
+# ADD ALLIANCE TO NORWENNA SPOUSE, OR TO CHARACTER THE SPOUSE IS PRIM HEIR TO
+
+	type = character_event
+	title = BTWK_story_event.0105.t
+	desc = BTWK_story_event.0105.desc
+	theme = realm 
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
 	}	
 	lower_left_portrait = {
-		character = character:5  # Mordred ap Mordred
+		character = character:5  # Mordred
 		animation = no
 	}
 	lower_center_portrait = {
-		character = character:2  # Merlin
+		character = character:14  # Tewdric
 		animation = no
 	}
 	lower_right_portrait = {
 		character = character:3  # Arthur
 		animation = no
 	}
-	
+
 	option = {
-		name = BTWK_story_event.0001.a
-		set_global_variable = utherillness
+		name = BTWK_story_event.0105.a
+		character:1 = {
+			trigger_event = { 
+				id = BTWK_story_event.0106
+				days = {25 50}
+			}
+		}
+	}
+}
+
+BTWK_story_event.0106 = {  
+	type = character_event
+	title = BTWK_story_event.0106.t
+	desc = BTWK_story_event.0106.desc
+	theme = skull 
+
+	# More fitting music cue/sounds needed
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
+	}	
+	option = {
+		name = BTWK_story_event.0106.a
 		character:1 = {
 			add_trait = pneumonic
-			add_character_flag = utherillness
-			if = {
-				limit = { NOT = { has_global_variable = utherincapable } }
-				trigger_event = {
-					id = BTWK_story_event.0002
-					days = {100 180}
-				}
+			# Need to remove the opinion from all characters affected - passed over for regent.
+			designate_diarch = character:51  # Father Bedwin 
```

Comment:
Would it make sense to put this in the history instead (or as well)?

Reply:
I placed it there in case the AI decided to change regent (based on the fact Bedwin is a lowborn). I also don't know how rigid regent succession on ruler succession is. I'll place it in the history files as well, I don't see how it can hurt.